packages/firebase_ai/firebase_ai/example/lib/utils/image_utils.dart

```diff
@@ -0,0 +1,140 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import 'package:firebase_ai/firebase_ai.dart';
+import 'package:image/image.dart' as img;
+import 'package:flutter/foundation.dart';
+
+// Helper class to pass parameters to the isolate
+class _IsolateParams {
+  final Uint8List imageBytes;
+  final ImagenDimensions newDimensions;
+  final ImagenImagePlacement newPosition;
+
+  _IsolateParams({
+    required Uint8List imageBytes,
+    required this.newDimensions,
+    required this.newPosition,
+  }) : imageBytes = Uint8List.fromList(imageBytes);
+}
+
+// Helper class to return results from the isolate
+class _IsolateResult {
+  final Uint8List paddedImageBytes;
+  final Uint8List maskBytes;
+
+  _IsolateResult({
+    required this.paddedImageBytes,
+    required this.maskBytes,
+  });
+}
+
+/// This is the top-level function that will run in the background isolate.
+/// It uses the 'image' package for all manipulations.
```

Comment:
[nit] Start method docs with a short verb-phrased header.

```suggestion
/// Processes the image request.
/// 
/// This is the top-level function that will run in the background isolate.
/// It uses the 'image' package for all manipulations.
```